packages/slate-editor/src/extensions/tables/TablesExtension.tsx

```diff
@@ -64,24 +65,26 @@ export function TablesExtension(): Extension {
             return undefined;
         },
         withOverrides: (editor) => {
-            return withTablesDeleteBehavior(
-                withTables(editor, {
-                    createContentNode: createParagraph,
-                    createTableNode: ({ children, ...props }) =>
-                        createTableNode({
-                            ...props,
-                            children: children as TableNode['children'] | undefined,
-                        }),
-                    createTableRowNode: ({ children, ...props }) =>
-                        createTableRowNode({
-                            ...props,
-                            children: children as TableRowNode['children'] | undefined,
-                        }),
-                    createTableCellNode,
-                    isTableNode,
-                    isTableRowNode,
-                    isTableCellNode,
-                }),
+            return withTablesCopyPasteBehavior(
+                withTablesDeleteBehavior(
+                    withTables(editor, {
```

Comment:
I guess it's time to use `flow()` to unfold these nesting levels.

See https://github.com/prezly/slate/blob/82ef1062880eca584a647e7bd3ac2019005c73b0/packages/slate-editor/src/modules/editor/createEditor.ts#L29-L44